pages/p/[productId].js

```diff
@@ -121,6 +133,25 @@ const Product = React.memo(lazyProps => {
     </Row>
   )
 
+  // Fetch variant data upon changing color or size options
+  useDidMountEffect(() => {
+    const url = URL.format({
+      pathname: `/api/p/${product.id}`,
+      query: {
+        color: get(color, 'id'),
+        size: get(size, 'id'),
```

Comment:
Same here "After implementing the get() || {} we can just use everywhere now color.id and so on, should not crash. "